logger/logger.go

```
@@ -21,18 +21,55 @@ const (
+const (
+	LevelDump = iota
+	LevelDebug
+	LevelInfo
+	LevelWarn
+	LevelError
+	LevelNone
+)
+
 var (
 	Logw         = Logf
 	logw         = Logf
```

golangci-lint (1.62.2)

var `logw` is unused

(unused)